v3/process/process_darwin.go

```diff
@@ -212,17 +210,18 @@ func (p *Process) GidsWithContext(ctx context.Context) ([]int32, error) {
 }
 
 func (p *Process) GroupsWithContext(ctx context.Context) ([]int32, error) {
-	k, err := p.getKProc()
-	if err != nil {
-		return nil, err
-	}
+	return nil, common.ErrNotImplementedError
```

Comment:
Sorry, I can not remember but is this `Groups` not working?

Reply:
I don't think it ever worked, see #913.

Reply:
thanks!